MyList.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+
+class MyList
+  include Enumerable
+  def initialize(*list)
+    @list = list
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+puts 'Test #all?'
+puts(list.my_all? { |n| n < 5 })
+puts '|n| n < 5 Should return true'
```

Comment:
- [ ] OPTIONAL
This line is returning _false_ instead of _true_. Kindly endeavor to fix it.